TemplateApplication/Contacts/Contacts.swift

```diff
@@ -10,8 +10,12 @@ import Foundation
 import SpeziContact
 import SwiftUI
 
```

Comment:
Please ensure that you have two empty lines between the import statements and the first lines of the swift files. Also applies to other files.